#!/usr/bin/env python3
PACKAGE = "robot_controllers"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("x_pick",   double_t,   0, "x value for pick position", 0.4  , -0.8 , 0.8)
gen.add("y_pick",   double_t,   0, "y value for pick position",  -0.25 , -0.8 , 0.8)
gen.add("z_pick",   double_t,   0, "z value for pick position",  0.088 , 0.0 , 0.6)

gen.add("x_place",   double_t,   0, "x value for place position", 0.4  , -0.8 , 0.8)
gen.add("y_place",   double_t,   0, "y value for place position",  -0.25 , -0.8 , 0.8)
gen.add("z_place",   double_t,   0, "z value for place position",  0.088 , 0.0 , 0.6)

gen.add("delta_height",   double_t,   0, "set how much higher than z_pick during move",  0.1 , 0.0 , 0.5)

exit(gen.generate(PACKAGE, "pickAndPlace", "PickAndPlace"))